.circleci/config.yml

```diff
@@ -52,6 +52,10 @@ jobs:
           name: Run rubocop
           command: |
             bundle exec rubocop
+      - run:
+          name: Run assets precompile
+          command: |
+            bundle exec rake assets:precompile
```

Comment:
Can't say I see a huge change in time here comparing https://circleci.com/gh/Safecast/safecastapi/882 to https://circleci.com/gh/Safecast/safecastapi/867

But probably not a bad idea to ensure precompile works as part of the build.

Reply:
I compared https://circleci.com/gh/Safecast/safecastapi/879 (Rspec 2:10) and https://circleci.com/gh/Safecast/safecastapi/882 (Rspec 0:51) 
I guess 879 was running without cache and took longer 🤔 
PS. Might be better to use system tests rather testing controller and views ;)

Reply:
Nice. TIL. This is the only rails app I work on and it's not on 5 yet, so I missed the news. We have a decent set of acceptance specs, but system tests look like they'll be a win when we can switch to them.